### ISSUE TITLE
Changing  Visibility via the Game Configuration Settings breaks server

### ISSUE THREAD

**Issue by CdrKeen** [mroote/factorio-server-manager#70]
if you change it from the config itself its fine, but if you look at the text field for visibility it displays `[object Object]` i think this is what is causing the issue. When you update the configuration it will no longer start the server and complain that there is an issue with the Syntax in Visibility.
![visibility](https://cloud.githubusercontent.com/assets/5512290/25094001/b19777ec-235a-11e7-8711-e6fb27feaa7a.PNG)

**Comment by sdobz** [comment]
Visibility changed from 'public'/'lan' to {'public':true/false, 'lan': true/false}

The docker version seems to overwrite this every time, I can't star the server

**Comment by zigarot** [comment]
yeah I'm getting this issue as well

**Comment by marchingphoenix** [comment]
found a workaround by specifically mapping the config file as read only via a -v option in my `docker run` command

```
docker create --name factorio-manager \
        --restart=always \
        --privileged=true \
        -e PGID=1000 -e PUID=1000  \
        -v /etc/localtime:/etc/localtime:ro \
        -v /docker/containers/factorio-manager/factorio/saves:/opt/factorio/saves \
        -v /docker/containers/factorio-manager/factorio/mods:/opt/factorio/mods \
        -v /docker/containers/factorio-manager/factorio/config:/opt/factorio/config \
        -v /docker/containers/factorio-manager/factorio/server-settings.json:/opt/factorio/config/server-settings.json:ro \
        -v /docker/containers/factorio-manager/security:/security \
        -p 443:443 -p 34197:34197/udp \
        majormjr/factorio-server-manager
```

**Comment by mroote** [comment]
fixed in 0.7.1
